deploy/GelatoCCTPReceiver.ts

```diff
@@ -0,0 +1,27 @@
+import { deployments, getChainId, getNamedAccounts } from "hardhat";
+import { DeployFunction } from "hardhat-deploy/types";
+import { ADDRESSES } from "../shared/constants";
+
+const name = "GelatoCCTPReceiver";
+
+const func: DeployFunction = async () => {
+  const { deploy } = deployments;
+  const { deployer } = await getNamedAccounts();
+
+  const chainId = await getChainId();
+  const address = ADDRESSES[chainId];
+
+  if (!address) throw new Error("Unsupported network");
+
+  const { usdc, messageTransmitter } = address;
+
+  await deploy(name, {
+    from: deployer,
+    args: [usdc, messageTransmitter],
+    log: true,
```

Comment:
Copy the logging etc. from https://github.com/gelatodigital/GEL-staking/blob/256d7ccab868e7d2af4d45c7d4fce70e83e349d4/deploy/GELStaking.deploy.ts#L15